junit-platform-commons/src/main/java/org/junit/platform/commons/util/ReflectionUtils.java

```diff
@@ -573,9 +573,11 @@ public static boolean isPackage(String packageName) {
 		Preconditions.notNull(clazz, "Class must not be null");
 		Preconditions.notNull(sortOrder, "MethodSortOrder must not be null");
 
-		// TODO [#333] Determine if we need to support bridged methods.
-
-		List<Method> localMethods = Arrays.asList(clazz.getDeclaredMethods());
+		// @formatter:off
+		List<Method> localMethods = Arrays.stream(clazz.getDeclaredMethods())
+				.filter(Method::isBridge) // [#333] don't collect bridge methods
```

Comment:
You are totally right! Look at the low number of tests processed ... and it didn't yield a big fat red light screaming: 🔴 HALT 🔴

Will add a test for this situation and revert to a former version, which I replaced with this inversed logic by error. It used to be:
```
List<Method> localMethods = Arrays.stream(clazz.getDeclaredMethods()).collect(toList());
localMethods.removeIf(Method::isBridge);
```